scripts/tests/test_validate_unwanted_patterns.py

```diff
@@ -0,0 +1,215 @@
+import io
+
+# the "as" is just so I won't have to change this in many places later.
+import validate_string_concatenation as validate_unwanted_patterns
+
+
+class TestBarePytestRaises:
+    def test_bare_pytest_raises(self):
+        fd = io.StringIO(
+            """
```

Comment:
May be using dedent makes the content of the files more readable?

Reply:
I got to be honest, I don't see how it adds more readability here, I know this was suggested for the code in b2d2d62 but in 796550e I don't think this is the case (IMO). can you please "re-review" (re-think) for the case in 796550e?